p2p/protocol/identify/grafana-dashboards/dashboard.json

```diff
@@ -0,0 +1,355 @@
+{
```

Comment:
Wdyt about moving this dashboard and others to a common top-level folder? So this could be `grafana-dashboards/identify/dashboard.json` and similarly `grafana-dashboards/swarm/swarm.json` and so on

This would make it easy to discover all of them at once and not make them buried inside each module.

Reply:
See https://github.com/libp2p/go-libp2p/issues/2055. I suggest doing that in a separate PR, and probably after the v0.25 release.

Reply:
good stuff, hadn't seen that.